### ISSUE TITLE
Does the pre-release version is now any update, from last one year?

### ISSUE THREAD

**Issue by diptilenka** [Microsoft/WinAppDriver#1830]
When we are planning to release the 1.3 version . from last 1 year it has been like this .

**Comment by pm90pl** [comment]
Yeah, it's extremely hard to put any trust into that tool when the support and communication are so poor.

**Comment by bigsteiny** [comment]
Agreed, I had the same question - this product is still linked to on Microsoft pages, but it's been over a year since last release? Has MS just abandoned this??

**Comment by pm90pl** [comment]
It's the Microsoft way of treating their technologies ;)

**Comment by pm90pl** [comment]
@DHowett any updates?

**Comment by DLightstone** [comment]
One can infer if someone at Microsoft is monitoring. Just post an appropriately disparaging comment.
If it is removed via some mechanism you have the "to busy to reply" answer
